RadixWallet/Features/AccountDetailsFeature/Coordinator/AccountDetails+Reducer.swift

```diff
@@ -88,7 +94,7 @@ public struct AccountDetails: Sendable, FeatureReducer {
 		case loadMnemonicResult(TaskResult<MnemonicWithPassphraseAndFactorSourceInfo>)
 
 		case loadImport
-		case loadProfileSnapshotForRecoverMnemonicsFlow(TaskResult<ProfileSnapshot>)
+		case loadControlledEntitiesForRecoverMnemonicsFlow(TaskResult<EntitiesControlledByFactorSource>)
```

Comment:
sure we can do this for now, important we do ABW-2206 soon, which moves import to Settings, never initated from any other screen than `Settings-> Account Settings & Security -> Seed Phrases`